x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/types.ts

```diff
@@ -156,6 +166,7 @@ export interface DefineStepRuleJson {
   machine_learning_job_id?: string[];
   saved_id?: string;
   query?: string;
+  data_view_id?: string | null;
```

Comment:
What's a `null` dataViewId reflect in contrast to an `undefined` one?

Is it when it gets wiped out within `DataViewSelector`?

``` 
// Most likely case here is that a data view of an existing rule was deleted and can no longer be found
```

Reply:
resolved here: f26eeae6e373fdf308c552d0b822cfe8050c7a3e